docs/en/tutorials/dataset_types.md

```diff
@@ -142,42 +189,89 @@ train = dict(
+We have support for reading annotation files from the full lmdb dataset (with images and annotations). It is now possible to read lmdb datasets commonly used in academia. We have also implemented a new dataset conversion tool, [recog2lmdb](https://github.com/open-mmlab/mmocr/blob/main/tools/data/utils/recog2lmdb.py). It converts the recognition dataset to lmdb format. See [PR982](https://github.com/open-mmlab/mmocr/pull/982) for more details.
+
+Here is an example configuration to load lmdb annotations:
 
-After that, you can use `LmdbLoader` in dataset like below.
 ```python
-img_prefix = 'tests/data/ocr_toy_dataset/imgs'
-train_anno_file = 'tests/data/ocr_toy_dataset/label.lmdb'
+lmdb_root = 'path to lmdb folder'
 train = dict(
-    type=dataset_type,
-    img_prefix=img_prefix,
-    ann_file=train_anno_file,
+    type='OCRDataset',
+    img_prefix=lmdb_root,
+    ann_file=lmdb_root,
     loader=dict(
-        type='LmdbLoader',
-        repeat=10,
+        type='AnnFileLoader',
+        repeat=1,
+        file_format='lmdb',
         parser=dict(
-            type='LineStrParser',
+            type='LineJsonParser',
             keys=['filename', 'text'],
             keys_idx=[0, 1],
             separator=' ')),
-    pipeline=train_pipeline,
+    pipeline=None,
     test_mode=False)
 ```
 
+#### Evaluation
+
+There are six evaluation metrics available for text recognition tasks: `word_acc`, `word_acc_ignore_case`, `word_acc_ignore_case_symbol`, `char_recall`, `char_precision` and `one_minus_ned`. See our [API doc](https://mmocr.readthedocs.io/en/latest/api.html#mmocr.core.evaluation.eval_ocr_metric) for explanations on metrics.
+
+By default, `OCRDataset` generates full reports on all the metrics if its evaluation
+metric is `acc`. Here is an example case for **training**.
+
+```python
+# Configuration
+evaluation = dict(interval=1, metric='acc')
+```
+
+```bash
+# Results
+{'0_char_recall': 0.0484, '0_char_precision': 0.6, '0_word_acc': 0.0, '0_word_acc_ignore_case': 0.0, '0_word_acc_ignore_case_symbol': 0.0, '0_1-N.E.D': 0.0525}
+```
+
+:::{note}
+'0_' prefixes result from `UniformConcatDataset`. It's kept here since MMOCR always wrap `UniformConcatDataset` around any datasets.
+:::
+
+If you want to conduct the evaluation on a subset of evaluation metrics:
+
+```python
+evaluation = dict(interval=1, metric=['word_acc_ignore_case', 'one_minus_ned'])
+```
+
+The result will look like:
+
+```bash
+{'0_word_acc_ignore_case': 0.0, '0_1-N.E.D': 0.0525}
+```
+
+**During testing**, you can specify the metrics to evaluate in the command line:
+
+```bash
+python tools/test.py configs/textrecog/crnn/crnn_toy_dataset.py crnn.pth --eval word_acc_ignore_case one_minus_ned
+```
+
 ### OCRSegDataset
 
 *Dataset for segmentation-based recognizer*
 
+It shares a similar architecture with `TextDetDataset`. Check out the [introduction](#textdetdataset) for details.
+
+
+#### Example Configuration
+
 ```python
 prefix = 'tests/data/ocr_char_ann_toy_dataset/'
```

Comment:
This example configuration still uses `HardDiskLoader`, which should be updated.